site/src/index.css

```
@@ -19,23 +19,35 @@ code {
 }
 
 :root {
-  --peterportal-primary-color-1: #2484c6;
-  --peterportal-primary-color-2: #74d1f6;
-  --peterportal-secondary-green: #81c284;
-  --peterportal-secondary-yellow: #f5d77f;
-  --peterportal-secondary-orange: #ecad6d;
-  --peterportal-secondary-red: #e7966d;
-  --peterportal-light-gray: #eeeeee;
-  --peterportal-mid-gray: #8d8d8d;
-  --peterportal-uci-gold: #ffd200;
-  --peterportal-gray-blue: #f5f6fc;
-  --zot-blue: var(--peterportal-primary-color-1);
-  --aldrich-blue: var(--peterportal-primary-color-2);
+  --blue-primary: #2484c6;
+  --blue-secondary: #5babe1;
+  --blue-tertiary: #a0ceee;
+  --blue-accent: #74d1f6;
+
+  --blue-primary-50-percent: #2484c680;
+  --blue-primary-80-percent: #2484c6cc;
+
+  --green-primary: green;
+  --rating-4: #81c284;
```

I think you should stick with one naming convention. I know we discussed both options, but I could have specified that it'd be better to be consistent between these
```suggestion
  --green-primary: green;
  --green-secondary: #81c284;
```
Because the rating-4 color is essentially a secondary green, we should name it as such if the primary one is going to be called `green-primary`

---

You would have to change this for other ratings as well, of course.
And for now, I do think it's fine if you change `--rating-3` to be `--yellow-secondary` and not have a `--yellow-primary`.

---

Agreed, fixed